library/core/src/ptr/mod.rs

```diff
@@ -95,24 +95,24 @@
 //!
 //! [valid value]: ../../reference/behavior-considered-undefined.html#invalid-values
 //!
-//! ## Allocated object
+//! ## Allocation
```

Comment:
This will break existing links to `#allocated-object`. Dos anyone have an idea how we could avoid that?

Reply:
Inline html? Something like:

```html
<a id="allocated-object"></a>
```

Reply:
> This will break existing links to `#allocated-object`. Does anyone have an idea how we could avoid that?

[Here's](https://github.com/rust-lang/reference/pull/1740/commits/8b5822b99b6a996e98a927f6da4283a0ddc48bab) what we do in the Reference.

Reply:
Yeah... but not sure if I can just put that in a doc comment.